python/MANIFEST.in

```diff
@@ -22,4 +22,5 @@ recursive-include deps/data *.data *.txt
 recursive-include deps/licenses *.txt
 recursive-include deps/examples *.py
 recursive-include lib *.zip
+recursive-include pyspark *.pyi py.typed
```

Comment:
__Note__:

Is it possible to do it through `package_data`, i.e.

```python
        package_data={
            '': ['*.pyi', 'py.typed'],
            ...
```

Doesn't seem to work here...